be/src/runtime/memory/mem_tracker_limiter.cpp

```diff
@@ -522,6 +530,15 @@ int64_t MemTrackerLimiter::free_top_overcommit_query(int64_t min_free_mem,
             profile, GCType::PROCESS);
 }
 
+int64_t MemTrackerLimiter::tg_free_top_overcommit_query(
```

Comment:
It seems unnecessary, just use `MemTrackerLimiter::free_top_memory_query` directly